src/helpers.js

```diff
@@ -53,7 +50,7 @@ const getHotkeyCallback = (keyMap, keyCode, eventKeyModifiers) => {
  * @param {Array} keyMap
  * @param {Object} modifiers Vue event modifiers
  */
-export const assignKeyHandler = (e, keyMap, modifiers) => {
+export const assignKeyHandler = (e, keyMap, modifiers, forbiddenNodes) => {
```

Comment:
```suggestion
export const assignKeyHandler = (e, keyMap, modifiers, forbiddenNodes = []) => {
```

So you key remove the `[]` in where you call `assignKeyHandler`

Reply:
I would rather set it to `DEFAULT_FORBIDDEN_NODES` since that's the default behavior that you had before. What do you say?